www/docusaurus.config.js

```diff
@@ -171,10 +195,14 @@ const config = {
                 label: "Twitter",
                 href: "https://twitter.com/firezonehq?utm_source=docs.firezone.dev",
               },
+              {
+                label: "LinkedIn",
+                href: "https://www.linkedin.com/company/firezonehq/",
+              },
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc.`,
+        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc. Wireguard® is a registered trademark of Jason A. Donenfeld.`,
```

Comment:
Properly spell WireGuard

```suggestion
        copyright: `Copyright © ${new Date().getFullYear()} Firezone, Inc. WireGuard® is a registered trademark of Jason A. Donenfeld.`,
```